### PR TITLE
navbar footer bugs

### LINKED ISSUE #192: Fix All Navbar & Footer Bugs
Fix all bugs on the [Find Your Shrek](https://docs.google.com/document/d/1f3dqxzRCiNGZ2yWU_FtyrNNjUOFZ8lxiEuXo958NVDM/edit?pli=1#heading=h.pylkt3kz9rjv) doc that are related to the navbar and footer.

### PR DESCRIPTION
**Pull request by ctc-devops**
Fix all bugs on the [Find Your Shrek](https://docs.google.com/document/d/1f3dqxzRCiNGZ2yWU_FtyrNNjUOFZ8lxiEuXo958NVDM/edit?pli=1#heading=h.pylkt3kz9rjv) doc that are related to the navbar and footer.
closes #192

### CODE REVIEW

**Comment by dgardezi** [comment]
fixed bugs: 4, 25, 44

File: src/components/NavBar/NavBar.module.css [hunk 1]
```diff
@@ -1,3 +1,9 @@
+.navbar {
+  position: sticky;
+  z-index: 900;
+  background-color: var(--color-white);
```
Comment (aliu55) [inline src/components/NavBar/NavBar.module.css:4]: Hmm the navbar is still missing a box shadow (Bug 4). You might just need to change the background color to something else.

**Review by aliu55 — CHANGES_REQUESTED** [review]
Good job on fixing the bugs :) please resolve the merge conflict!

**Review by aliu55 — APPROVED** [review]
LGTM

**Review by sylvesteryue — APPROVED** [review]
LGTM

**Pull request merged**